src/pages/UnlockedWallet/OverviewPage/AmountsOverviewPanel.tsx

```diff
@@ -174,20 +175,24 @@ const UnlockedWalletPanelStyled = styled(UnlockedWalletPanel)`
   position: relative;
 `
 
+const BackgroundGradient = styled.div`
+  position: absolute;
+  top: -100px;
+  bottom: -150px;
+  right: 0;
+  left: 0;
+  background: linear-gradient(${({ theme }) => theme.bg.primary}, transparent);
+  pointer-events: none;
+`
+
 const Panel = styled.div<{ worth?: number }>`
+  position: relative;
   display: flex;
   gap: 30px;
   align-items: center;
 
   margin-bottom: 45px;
   padding: 30px 0;
-
-  ${({ worth }) =>
-    !worth &&
-    css`
-      position: relative;
-      z-index: 1;
-    `}
```

Comment:
This bugged me for a little while. Do you remember what it was for @nop33 ?

Reply:
Related to hovering the chart